src/searchengine.qrc

```diff
@@ -1,16 +1,13 @@
 <RCC>
     <qresource prefix="/">
-        <file>searchengine/nova3/helpers.py</file>
-        <file>searchengine/nova3/nova2.py</file>
-        <file>searchengine/nova3/nova2dl.py</file>
-        <file>searchengine/nova3/novaprinter.py</file>
-        <file>searchengine/nova3/sgmllib3.py</file>
-        <file>searchengine/nova3/socks.py</file>
-        <file>searchengine/nova/fix_encoding.py</file>
-        <file>searchengine/nova/helpers.py</file>
-        <file>searchengine/nova/nova2.py</file>
-        <file>searchengine/nova/nova2dl.py</file>
-        <file>searchengine/nova/novaprinter.py</file>
-        <file>searchengine/nova/socks.py</file>
+        <file>searchengine/nova6/nova2.py</file>
+        <file>searchengine/nova6/nova2dl.py</file>
```

Comment:
@Depaulicious:  Are these `nova2` files supposed to still be here?  

I see the equivalent `nova6` files directly below

Reply:
They are wrapper scripts to avoid depending on setuptools.